scalr/resource_scalr_endpoint.go

```diff
@@ -0,0 +1,179 @@
+package scalr
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	scalr "github.com/scalr/go-scalr"
+)
+
+func resourceScalrEndpoint() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceScalrEndpointCreate,
+		Read:   resourceScalrEndpointRead,
+		Update: resourceScalrEndpointUpdate,
+		Delete: resourceScalrEndpointDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"http_method": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"max_attempts": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+			},
+
+			"url": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"secret_key": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
We need to add `Sensitive: true` - https://www.terraform.io/docs/extend/best-practices/sensitive-state.html. See https://github.com/Scalr/terraform-provider-scalr/blob/develop/scalr/resource_scalr_variable.go#L42

Reply:
discussed separately, will add Sensitive